### DISCUSSION TITLE
Setting up a domain

### DISCUSSION THREAD

**Discussion by CWM030** [simple-login/app#1923] [Q&A]
I've just signed up for SimpleLogin

But, when adding the records, SL is erroring out saying its seeing Fastmail's Records..

WELL YEAH, That has to be there for me to recive emails at my domain!

I am confused, like, how the H do I make it to where both can co-exist and work? lol

TIA!

**Comment by CWM030** [comment]
I just tried again to set my domain up and I can't pass the domain verification. 

I sat up last night mail.domain.tld

but SL says that it can't find the TXT records. 

I evem changed the TTL to 1 min!!! Nada.

**Comment by CWM030** [comment]
This is interesting... 

The only way I could get my domain to pass verification was by adding only SimpleLogin's MX records
then setting up a alias in SimpleLogin as FirstName@DOMAIN.TLD and SL forwards my emails to the domain? So I don't need Fastmail's MX records at all? 

What about Fastmail's SPF, DMARC, and DKIM records, do I need to keep those set up, since I use smtp.fastmail.com to send emails?

I'm confused.

Thanks.

**Comment by ghost** [comment]
Hi.

Hope this will help:
1. Create SimpleLogin account on that mailbox on which you want to receive forwarded messages from your SL aliases.
2. Remove ALL information from your DNS records in custom domain.
3. In SL custom domain settings follow instructions to add requested information. Make sure NOT to put 2 records simultaneously.

This should help. Secondly I STRONGLY recommend to set 2FA in settings.

P.S: Next time mail hi@simplelogin.io or support@simplelogin.zendesk.com. Here they reply hardly ever. Or you can tag `@nguyenkims`
